packages/date-picker/src/DatePicker/CalendarNav/CalendarNav.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import { Icon } from "@kaizen/component-library"
+import arrowRight from "@kaizen/component-library/icons/arrow-right.icon.svg"
+import arrowLeft from "@kaizen/component-library/icons/arrow-left.icon.svg"
+import { NavbarElementProps } from "react-day-picker/types/Props"
+import styles from "../DatePicker.scss"
+
+export type CalendarNavProps = Pick<
+  NavbarElementProps,
+  "onPreviousClick" | "onNextClick"
+>
+
+export const CalendarNav: React.VFC<CalendarNavProps> = ({
+  onPreviousClick,
+  onNextClick,
+}) => (
+  <div className={styles.navbar}>
+    <button
```

Comment:
Out of curiosity, did you explore using an `IconButton` here instead of a custom button + `Icon`?

Reply:
Yep we did look at that too start with, but it's too big and design didn't want a super small version in the UI kit.